src/Jackett.Common/Indexers/Libble.cs

```diff
@@ -0,0 +1,321 @@
+                            {
+                                release.PublishDate = DateTime.ParseExact(
+                                    releaseDateDetails.GetAttribute("title").Trim(),
+                                    "MMM dd yyyy, HH:mm",
+                                    CultureInfo.InvariantCulture,
+                                    DateTimeStyles.AssumeUniversal
+                                );
+                            }
+                            catch (Exception)
+                            {
+                            }
+
+                            release.Files = ParseUtil.CoerceInt(releaseFileCountDetails.TextContent.Trim());
+                            release.Grabs = ParseUtil.CoerceInt(releaseGrabsDetails.TextContent.Trim());
+                            release.Seeders = ParseUtil.CoerceInt(releaseSeedsCountDetails.TextContent.Trim());
+                            release.Peers = release.Seeders + ParseUtil.CoerceInt(releasePeersCountDetails.TextContent.Trim());
+                            release.Size = ReleaseInfo.GetBytes(releaseSizeDetails.TextContent.Trim());
+                            release.BannerUrl = releaseThumbnailUri;
+                            release.Category = releaseNewznabCategory;
+                            release.MinimumSeedTime = 259200; // 72 hours
+
+                            // Attempt to find volume factor tag
+                            release.DownloadVolumeFactor = 1;
+                            release.UploadVolumeFactor = 1;
+                            var releaseTags = releaseMediaType.Split('/').Select(tag => tag.Trim()).ToList();
+                            for (int i = releaseTags.Count - 1; i >= 0; i--)
+                            {
+                                string releaseTag = releaseTags[i];
+                                if (VolumeTagMappings.ContainsKey(releaseTag)) {
+                                    VolumeFactorTag volumeFactor = VolumeTagMappings[releaseTag];
+                                    release.DownloadVolumeFactor = volumeFactor.DownloadVolumeFactor;
+                                    release.UploadVolumeFactor = volumeFactor.UploadVolumeFactor;
+                                    releaseTags.RemoveAt(i);
+                                }
+                            }
+
+                            // Set title (with volume factor tags stripped)
+                            var releaseTagsString = string.Join(" / ", releaseTags);
+                            release.Title = String.Format("{0} - {1} [{2}] {3}", releaseArtist, releaseAlbumName, releaseAlbumYear, releaseTagsString);
+
+                            releases.Add(release);
+                        }
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                OnParseError(searchPage.Content, ex);
+            }
+
+            if (!CookieHeader.Trim().Equals(prevCook.Trim()))
```

Comment:
I think it's better to not play with cookies. Jackett will do it for your.
Try to remove all cookie related code in this PR and test that everything is still working.

Reply:
Sounds good, sorry! This was initially copy/pasted from the XSpeeds tracker (as that was another one I had access to and could test).